### PR TITLE
getting-started: Added remark about cloning repo

### PR DESCRIPTION
**Pull request by cinnamon-coder-hub**
Adding to #774

### CODE REVIEW

File: documentation/devonfw-ide-contribution-getting-started.asciidoc [hunk 1]
```diff
@@ -5,12 +5,12 @@ toc::[]
...
 
 == Contribution
 To begin, read through the https://github.com/devonfw/.github/blob/master/CONTRIBUTING.asciidoc[Contribution Guidelines], which you should be sure to follow.
-First steps regarding forks or cloning of repos and creating branches, as well as some git commands, https://github.com/firstcontributions/first-contributions[can be found here].
-Clone the https://github.com/devonfw/ide[project IDE] at the beginning.
+First steps regarding forks or cloning of repos and creating branches, as well as some git commands, can be found inside https://github.com/firstcontributions/first-contributions[ this first contributions guideline].
+ To contribute code or text regarding the devon-ide at first you need to fork the https://github.com/devonfw/ide[project IDE] and can then clone your fork into your devon ide installations workspaces-folder (just make sure to execute 'git init' inside 'workspaces' before cloning). Here the https://github.com/firstcontributions/first-contributions[first contributions guideline] can be very helpful.
```
Comment (tobka777) [inline documentation/devonfw-ide-contribution-getting-started.asciidoc:13]: Why should execute `git init`? That makes no sense to me.

**Review by tobka777 — CHANGES_REQUESTED** [review]
Thank you for your addition. Look at my comment.

File: documentation/devonfw-ide-contribution-getting-started.asciidoc (hunk 1 above)
Comment (tobka777) [reply documentation/devonfw-ide-contribution-getting-started.asciidoc]: maybe write it like this: 
and then you can clone your fork into the folder `workspaces` in your devon-ide installation

Comment (cinnamon-coder-hub) [reply documentation/devonfw-ide-contribution-getting-started.asciidoc]: As it turned out, this was a kind of bug or eventually some config issue. In the folder I tested the clone process the clone command worked, but after executing `git init`. In other folders `git clone` seems to do just fine without `git init`.

**Review by tobka777 — APPROVED** [review]

**Review by hohwille — APPROVED** [review]
@cinnamon-coder-hub thanks for your PR and improvement 👍 
I went through the changes and took the opportunity to add several improvements and commands to setup git, fork, upstream, etc. to get started. It would be great if someone could again have a quick look if that makes sense and reflects the way we are working.

**Comment by hohwille** [comment]
As nobody left a remark, I decided to merge this.
Everybody is still invited to have a look and contribute further improvements.

**Comment by cinnamon-coder-hub** [comment]
> @cinnamon-coder-hub thanks for your PR and improvement 👍 I went through the changes and took the opportunity to add several improvements and commands to setup git, fork, upstream, etc. to get started. It would be great if someone could again have a quick look if that makes sense and reflects the way we are working.

@hohwille Especially regarding the contribution-section I think that this will be of great help for newbies not used to git(hub) and our projects structure, as I often felt that newbies where confused or struggled to memorize all the details about our working process together with other information all at once (what I can totally understand). It was not undoable for them, but just took them more time and effort then necessary. So I think that this rather detailed contribution description is justified and also helps advanced developers to safe some time for beginners introductions, as this topic popped up often.

**Pull request merged**